processes-webapp/src/main/webapp/vue-app/processes/components/Processes.vue

```diff
@@ -356,20 +366,19 @@ export default {
       this.alert = true;
       window.setTimeout(() => this.alert = false, 5000);
     },
-
-    
     addNewWorkFlow(workflow) {
       this.saving = true;
       this.$processesService.addNewWorkFlow(workflow).then(workflow => {
         if (workflow){
           this.$root.$emit('workflow-added', {workflow: workflow, filter: this.enabled});
           this.displayMessage({type: 'success', message: this.$t('processes.workflow.add.success.message')});
+          this.showFilter();
```

Comment:
we need to keep the call to showFilter when adding a process to show the filter in the case where there are no processes and we add new one